src/chapters/unit-testing/hello-jasmine.rst

```diff
@@ -0,0 +1,301 @@
+Hello, Jasmine!
+===============
+
+.. index:: ! Jasmine
+
+In order to unit test our code, we need to use a module. Such a module of called a **unit-testing framework**, **test runner**, or **test harnesss**, and there are `many to choose from <https://en.wikipedia.org/wiki/List_of_unit_testing_frameworks#JavaScript>`_. 
+
+We will use `Jasmine <https://jasmine.github.io/>`_, a popular JavaScript testing framework. 
+
+Using Jasmine
+-------------
+
+.. index:: 
+   single: Jasmine; installing
+
+Installing Jasmine with NPM is simple:
```

Comment:
They don't. Shortly after it states that you don't have to do this. Should we move that to be right after npm is mentioned?

Reply:
I don't see where it states that npm is an optional install.  Regardless, I can see the students responding to "Installing with NPM" with confusion, since there is no mention of it before. I remember installing npm and Jasmine to make testing run in VSCode, but if the students do not need to install Jasmine on their machines, then avoid complicating the issue with npm.

If the students do need Jasmine, then install with something they have already used, like pip or conda, or provide a link to complete instructions (maybe in an appendix).